rel/files/mongooseim.cfg

```diff
@@ -441,15 +441,28 @@
 {{{rdbms_server}}}
 %%
 %% If you want to specify the port:
-%%{rdbms_server, {mysql, "server", 1234, "database", "username", "password"}}.
+%%{outgoing_pools, [
+%%  {rdbms, global, default, [{workers, 10}],
+%%   [{server, {mysql, "server", 1234, "database", "username", "password"}}]}]}.
+
+%% Interval to make a dummy SQL request to keep the connections to the
+%% database alive. Specify in seconds: for example 28800 means 8 hours
+%%{outgoing_pools, [
+%%  {rdbms, global, default, [{workers, 10}],
+%%   [{server, {mysql, "server", 1234, "database", "username", "password"}},
+%%    {keepalive_interval, 10}]}]}.
 
 %%
 %% PostgreSQL server:
 %%
-%%{rdbms_server, {pgsql, "server", "database", "username", "password"}}.
+%%{outgoing_pools, [
```

Comment:
There can be only one `outgoing_pools` config option. That's why I suggest to group all the examples inside one `outgoing_pools`. Otherwise people may start uncommenting the examples in random error and will be surprised that sth doesn't work. Other, probably even better option would be to merge all the `outgoing_pools` into one when the config is read from the file.

Reply:
I'm thinking of leaving this one as is when it comes to the configuration because putting all examples into a single `outgoing_pools` option will make the templating part (`{{rdbms_server}}`) more awkward.

I like the idea about merging pools.

Reply:
I think it's already awkward :) In the config file there are several commented out examples for different DBs and only one {{{rdbms_server}}}. I think it would still be as awkward as before :)